src/main/java/org/thoughtcrime/securesms/scribbles/UriGlideRenderer.java

```diff
@@ -303,7 +303,7 @@ public int describeContents() {
 
   @Override
   public void writeToParcel(Parcel dest, int flags) {
-    dest.writeString(imageUri.toString());
+    dest.writeString(imageUri!=null? imageUri.toString() : "");
```

Comment:
tbh, not sure what that Parcel thing is and if it is better to write to parcel in this case or just "do nothing" in this case

Reply:
"Parcel" is used for serialising and unserializing things, usually for IPC or so, https://developer.android.com/reference/android/os/Parcel

not totally sure for what it is needed here, might be some relict of Signal code.

in any case, this fixes as crash :)